book/appendix/wallaroo-command-line-options.md

```diff
@@ -4,66 +4,100 @@ Every Wallaroo option exposes a set of command-line options that are used to con
 
 ## Command-Line Parameters
 
-When running a Wallaroo application binary, we use some of the following command line parameters (a star indicates it is required, a plus that it is required for multi-worker runs):
+When running a Wallaroo application, we use some of the following command line parameters (a star indicates it is required, a plus that it is required for multi-worker runs):
 
-```
-      --in/-i *[Comma-separated list of input addresses sources listen on]
-      --out/-o *[Sets address for sink outputs]
-      --control/-c +[Sets address for control channel]
-      --data/-d +[Sets address for data channel]
-      --worker-count/-w +[Sets total number of workers, including topology
-        initializer]
-      --worker-name/-n +[Sets name for this worker]
-      --metrics/-m [Sets address for external metrics (e.g. monitoring hub)]
-      --cluster-initializer/-t [Flag that sets this process as the
-        initializer]
-      --resilience-dir/-r [Sets directory to write resilience files to,
-        e.g. -r /tmp/data (no trailing slash)]
-      --ponythreads [Number of application threads. Used as part of a high-performance configuration.]
-      --ponypinasio [Used as part of a high-performance configuration.]
-      --ponynoblock [Used as part of a high-performance configuration.]
+```bash
+  --in/-i *[Comma-separated list of input addresses sources listen on]
+  --control/-c +[Sets address for initializer control channel; sets
+    control address to connect to for non-initializers]
+  --data/-d +[Sets address for initializer data channel]
+  --my-control [Optionally sets address for my data channel]
+  --my-data [Optionally sets address for my data channel]
+  --phone-home/-p [Sets address for phone home connection]
+  --external/-e [Sets address for external message channel]
+  --worker-count/-w +[Sets cluster initialier's total number of workers,
+    including cluster initializer itself]
+  --name/-n +[Sets name for this worker]
+
+  --metrics/-m [Sets address for external metrics (e.g. monitoring hub)]
+  --cluster-initializer/-t [Sets this process as the cluster
+    initializing process (that status is meaningless after init is done)]
+  --resilience-dir/-r [Sets directory to write resilience files to,
+    e.g. -r /tmp/data (no trailing slash)]
+  --log-rotation [Enables log rotation. Default: off]
+  --event-log-file-size/-l [Optionally set a file size for triggering
+    event log file rotation. If no file size is set, log rotation is only
+    triggered by external control messages sent to the address used with
+    --external]
+
+  --join/j [When a new worker is joining a running cluster, pass the
+    control channel address of any worker as the value for this
+    parameter]
+  --stop-world/u [Sets pause before state migration after the stop the
+    world]
+
+-------------------------------------------------------------------------
+To set Wallaroo modes, compile with the following -D flags:
```

Comment:
why is this in here?  this is not a command line parameter. its a build time option. this shouldn't be here. please remove